app/components/Browser/Browser.jsx

```diff
@@ -64,11 +73,7 @@ class Browser extends Component
 
         const theBrowser = this;
 
-        //jest/electron workaround as no remote in non-render process
-        const currentWebContentsId = remote ? remote.getCurrentWebContents().id : 1;
-
-        // this is mounted but its not show?
-        this.setState( { windowId: currentWebContentsId } );
+        
```

Comment:
Again cosmetic, but important to keep the code clean and readable, perhaps remove one of the two empty lines?